microcosm_flask/paging.py

```diff
@@ -103,11 +114,23 @@ def to_dict(self):
                 self.schema.dump(item).data if self.schema else item
                 for item in self.items
             ],
-            _links=self.links.to_dict(),
+            _links=self._links,
```

Comment:
The `to_dict()` method is basically deprecated. I haven't checked to see whether we use it anywhere else.

Instead, we're just passing this `PaginatedList` object to our schema. The main limitation here is that the values within `self.page.to_dict()` has to be accessed via `self.page.foo`; if we want these returned in the paging result, we need to subclass the paging schema and give it extra fields with these mappings.